### DISCUSSION TITLE
Is this repo still maintained?

### DISCUSSION THREAD

**Discussion by e-t-l** [wrwrabbit/Partisan-SMS#73] [Q&A]
I see that the last release was published almost a year ago, and I don't see any development branches with a commit within the last 2 years. Is P-SMS still being developed/maintained? (g-d knows there were enough unresolved bugs in the original QKSMS...)

**Comment by inson1** [comment]
last commit here was 9 months ago, so I dont think so

Plus this https://github.com/wrwrabbit/Partisan-SMS/issues/34#issuecomment-1704289586
